src/uwloc/data/main.py

```diff
@@ -61,13 +66,11 @@ def _find_start_date(wavs_dir: str) -> datetime:
     files = sorted(glob.iglob(os.path.join(wavs_dir, WAV_PATTERN), recursive=True))
     for f in files:
         try:
-            (_, timestamp) = read_wav_metadata(f)
+            _, timestamp = read_wav_metadata(f)
             start_date = min(timestamp, start_date)
         except Exception as e:
-            typer.echo(f"Warning: Failed to read {f}. {e}", err=True)
+            logger.warning(f"Warning: Failed to read {f}. {e}")
```

Comment:
Same here, should be `ERROR` level as mentioned above.

Reply:
I'll keep this as a warning since it's a scan of all the files to find the min date. If there's a bad file in there it won't necessarily alter the result, but good to warn about.